tests/std/tests/P1007R3_assume_aligned/test.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <memory>
+
+static void can_change_value_via_assume_aligned(int* p) {

```

Comment:
As this is a single-source-file test, `static` doesn't do anything; I recommend removing it from these functions.